### PR TITLE
BJMIR-632 Add tournament details to match live feed snapshot

### PR DESCRIPTION
**Pull request by mkingori**

### CODE REVIEW

**Review by melias122 — APPROVED** [review]

**Comment by melias122** [comment]
Before merge, wait for 2 approval

**Comment by mkingori** [comment]
@pmachan please review this PR.

File: proto/bragi/cs2.proto [hunk 1]
```diff
@@ -42,6 +42,7 @@ message CS2MatchSnapshot {
   DataStatus data_status = 6;
   repeated Announcement announcements = 4;
   CS2MatchState match_state = 5;
+  bragi.Tournament tournament  = 7;
```
Comment (kafkapre) [inline proto/bragi/cs2.proto:45]: `Tournament tournament = 7;` should be enough.

Occurs multiple times

Comment (mkingori) [reply proto/bragi/cs2.proto]: Fixed

**Review by kafkapre — APPROVED** [review]

**Comment by melias122** [comment]
Appved @mkingori I think it will need another regenerate

**Comment by mkingori** [comment]
> Appved @mkingori I think it will need another regenerate

There were no new changes that needed regenerating

**Pull request merged**